packages_dev/dev/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@decidim/dev",
+  "description": "The dev dependencies for Decidim",
+  "version": "0.25.0-dev",
+  "repository": {
+    "url": "git@github.com:decidim/decidim.git",
+    "type": "git",
+    "directory": "packages/dev"
+  },
+  "author": "Decidim Contributors",
+  "license": "AGPL-3.0",
+  "engines": {
+    "node": "^15.14.0",
+    "npm": "^7.7.2"
+  },
+  "dependencies": {
+    "@decidim/eslint-config": "https://gitpkg.now.sh/mainio/decidim/packages/eslint-config?feature/split-npm-packages",
```

Comment:
💬 Same here with the branch reference

Reply:
Yes, after this is merged (in another PR).